### PR TITLE
Add testing Servlet files to return User's and Competition's information.

### PR DESCRIPTION
**Pull request by Phoebe-Khokgawe**
The information that's return is hard coded for now just to test that the Servlet is working.

### CODE REVIEW

**Comment by Phoebe-Khokgawe** [comment]
@googlebot I consent.

File: project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
```
Comment (roonil-wazlib) [inline project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java:1]: 2021?

File: project/src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (roonil-wazlib) [inline project/src/main/webapp/index.html:1]: This file is junk now right?

**Review by roonil-wazlib — CHANGES_REQUESTED** [review]

File: project/src/main/webapp/index.html (hunk 2 above)
Comment (Phoebe-Khokgawe) [reply project/src/main/webapp/index.html]: Yes you are right! f90372ec688566aba6c650a6f740a1dab6477c1a

File: project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java (hunk 1 above)
Comment (Phoebe-Khokgawe) [reply project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java]: fix f90372ec688566aba6c650a6f740a1dab6477c1a

**Comment by roonil-wazlib** [comment]
@googlebot I consent.

**Review by roonil-wazlib — APPROVED** [review]

**Review by texm — APPROVED** [review]

**Pull request merged**